.github/workflows/ci.yml

```diff
@@ -73,6 +73,7 @@ jobs:
   audit:
     runs-on: ubuntu-latest
     name: Audit
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
```

Comment:
What do you think of extracting the audit to another file and using the "on master pull_request" option? I think we could avoid the weird if by doing that

Reply:
Sure, I'll make the change